m4_sessions/1808-inning/machine_learning.md

```diff
@@ -1,20 +1,24 @@
 ## Group Members:
-
+Ryan Harrington
+David Dunn
+Tyler Wesley
 
 
 ## Name of Session: 
+Machine Learning Blessing or Curse
```

Comment:
Give me a little more..."Come to this session to discuss the advantages and dangers of ML."